src/Task/Task.php

```
@@ -88,15 +88,13 @@ public function run(Context $context): void
 
-        try {
-            call_user_func($this->callback); // call task
-        } finally {
-            if ($context->getConfig() !== null) {
-                $context->getConfig()->set('working_path', null);
-            }
-
-            Context::pop();
+        call_user_func($this->callback); // call task
+        
+        if ($context->getConfig() !== null) {
```

PHPStan (2.1.15)

Strict comparison using !== between Deployer\Configuration and null will always evaluate to true.


(notIdentical.alwaysTrue)